src/pet.js

```diff
@@ -70,16 +70,21 @@ function Pet() {
     },
```

Comment:
In your growUp() method, you've checked the pet's fitness before you do any changes, which is brilliant. Why didn't you check the hunger? And does the hunger have a maximum value?